news/2025/2025-04-15-the-followpoint-azer-the-tournament-titan.md

```diff
@@ -0,0 +1,305 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+How did you get introduced to osu! and what was your first impression of it?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+A friend I played Counter Strike with told me the game is great for aim training (classic...), and since I have a heavy music background I picked it up pretty quickly. It felt extremely foreign at first since I had never been introduced to otaku culture prior and was kinda fascinated to learn about so many artists and new music genres I had never heard before. I had played a lot of Guitar Hero but never really got to hear any eastern music or play any other rhythm games, so it was really captivating. Some of the artists I initially discovered here when I was new are still some of my favourites to this day. I wasn't particularly dragged in by watching any gameplay or notable players, but mostly just to the unique experience it provides.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Is there a specific memory from osu! that stands out to you? If so, which one?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2155578" style="background-image: url('/wiki/shared/avatars/Azer.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/2155578">Azer</a></p>
+
+It's hard to choose one, since I've been in this community for almost half of my entire life now, but if I had to choose one really early on that I remember very fondly, it was my [very first match in OWC 2013](https://www.youtube.com/watch?v=tCaXKs4z5TU). I had only been playing for a year, and our team (Canada) was not particularly high level or expected to win anything. Our first match in the group stage just happened to be against the strongest team in the tournament - Japan.
```

Comment:
```suggestion
It's hard to choose one, since I've been in this community for almost half my life now, but if I had to choose one really early on that I remember very fondly, it was my [very first match in OWC 2013](https://www.youtube.com/watch?v=tCaXKs4z5TU). I had only been playing for a year, and our team (Canada) was not particularly high level or expected to win anything. Our first match in the group stage just happened to be against the strongest team in the tournament - Japan.
```